lib/roo_on_rails/rack/populate_env_from_jwt.rb

```diff
@@ -41,7 +41,6 @@ def call(env)
       rescue UnacceptableKeyError, JSON::JWT::Exception => e
         # Identifying user is clearly attempting to hack or has been given a totally incorrect
         # token, log this and flag as Forbidden, without executing the rest of the middleware stack.
-        ::NewRelic::Agent.notice_error(e) if defined?(NewRelic)
```

Comment:
Would it be worth logging this with Rails.logger and Raven as below?

Reply:
Worth logging with Raven - but Rails isn't in scope here so we can't use logger